### ISSUE TITLE
ember - resolve decorators warnings

### ISSUE THREAD

**Issue by kaden-weber** [ManhanGroup/massbuilds#5]
The project makes use of decorators, but so far removing the extra packages that used to enable them causes some obvious issues. It's possible that the decorators don't fit the spec mentioned in the warnings.

```
WARNING: ember-decorators (used in calbuilds): You have not installed @ember-decorators/babel-transforms. It has been extracted to a separate addon. See instructions for installation: https://github.com/ember-decorators/babel-transforms#readme
WARNING: @ember-decorators/babel-transforms: You are using an unsupported ember-cli-babel version, decorator/class-property transforms will not be included automatically
WARNING: calbuilds has added the decorators plugin to its build, but ember-cli-babel provides these by default now! You can remove the transforms, or the addon that provided them, such as @ember-decorators/babel-transforms. Ember supports the stage 1 decorator spec and transforms, so if you were using stage 2, you'll need to ensure that your decorators are compatible, or convert them to stage 1.
```

Based on these, the first step would be updating the ember-cli-babel version

**Comment by kaden-weber** [comment]
something like this looks it should be resolved before upgrading, but I believe that decorator plugin related problems can be best solved by getting to ember 3.4, where decorators begin to be shipped with ember.

**Comment by kaden-weber** [comment]
Done on #10
